### PR TITLE
DTSPB-2319: PP: Ensure death certificate isn't appearing in cover she…

### PR DESCRIPTION
**Pull request by srochani**
…et/complete application page/send docs on case progress

**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ x] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/DTSPB-2319

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/probate/service/ConfirmationResponseService.java [hunk 1]
```diff
@@ -275,7 +275,6 @@ private TemplateResponse generateNextStepsBodyMarkdown(CCDData ccdData) {
         keyValue.put("{{ihtForm}}", getIhtForm(ccdData));
         keyValue.put("{{additionalInfo}}", additionalInfo);
         keyValue.put("{{renouncingExecutors}}", getRenouncingExecutors(ccdData.getExecutors()));
-        keyValue.put("{{deadExecutors}}", getDeadExecutors(ccdData.getExecutors()));
```
Comment (srochani) [inline src/main/java/uk/gov/hmcts/probate/service/ConfirmationResponseService.java:276]: deadExecutors key was used only for deathAfter and deathBefore so removed from markdown

File: src/test/java/uk/gov/hmcts/probate/service/ConfirmationResponseServiceTest.java [hunk 2]
```diff
@@ -810,6 +810,33 @@ public void shouldGetNextStepsConfirmationWithNoPayment() {
         assertEquals("0.00", nextStepsValues.get("{{paymentAmount}}"));
     }
 
+    @Test
```
Comment (sanjayparekh-hmcts) [inline src/test/java/uk/gov/hmcts/probate/service/ConfirmationResponseServiceTest.java:882]: There is also a confirmationServiceFeatureTest - does that need updating? I see tests on there for dead execs

**Review by sanjayparekh-hmcts — CHANGES_REQUESTED** [review]

File: src/test/java/uk/gov/hmcts/probate/service/ConfirmationResponseServiceTest.java (hunk 2 above)
Comment (srochani) [reply src/test/java/uk/gov/hmcts/probate/service/ConfirmationResponseServiceTest.java:882]: Yes, I have already updated the output of following tests of ConfirmationResponseServiceFeatureTest
shouldGenerateCorrectConfirmationBodyWithAllCombinationsForAdditionalOptionsAndMultiples, shouldGenerateCorrectConfirmationBodyWithDeadExecutor, shouldGenerateCorrectConfirmationBodyWithAllCombinationsForAdditionalOptions and shouldGenerateCorrectConfirmationBodyWithMultipleDeadExecutors

Comment (sanjayparekh-hmcts) [reply src/test/java/uk/gov/hmcts/probate/service/ConfirmationResponseServiceTest.java:882]: OK - thanks

**Review by sanjayparekh-hmcts — APPROVED** [review]

**Pull request merged**